### PR TITLE
Update APIClient for plugin improvement changes - Closes #6685, #6703 and #6620

### LINKED ISSUE #6620: Update all the functional tests after message filtering changes
### Description

After introducing message filtering and zeroMQ usage, some of the functional tests might fail which needs to be fixed

### Motivation

- All `test/functional` tests should pass

### Additional Information

- Related #6612

### LINKED ISSUE #6685: Update IPCClient in Lisk API Client to use zeromq based channel support
### Description

Based on the changes made in https://github.com/LiskHQ/lisk-sdk/issues/6612. PM2-axon is replaced with zeromq that brings some changes in how api client is created which needs to be changed for IPCClient in `@liskhq/lisk-api-client`

### Acceptance Criteria

- All tests related to IPC Client should pass
- Should be able to create an IPC client and able to connect and interact with a running node

### Additional Information

Related #6612

### LINKED ISSUE #6703: Update wsclient on lisk-api-client
### Description

WS client on APIClient should be updated to include `subscribe` and `unsubscribe` added in #6617 

### Acceptance Criteria

- WSClient is working with the updated framework ws RPC endpoint
- Tests are updated

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #6685 and resolves #6703 resolves #6620

### How was it solved?

- Update IPCClient to use zeromq
- Update WSClient to support subscribe/unsubscribe
- Disable functional test due to instability of zeromq on jest
- Fix minor bugs

### How was it tested?

- All test should be green now
- Run test app and use APIClient (IPC / WS) to connect to the node

### CODE REVIEW

File: framework/src/controller/ws/ws_server.ts [hunk 1]
```diff
@@ -41,11 +41,11 @@ export class WSServer {
 		this._logger = options.logger;
 	}
 
-	public start(
-		registeredEvents: ReadonlyArray<string>,
-		messageHandler: WSMessageHandler,
-	): WebSocket.Server {
-		this._registeredEvents = registeredEvents;
+	public registerAllowedEvent(events: string[]) {
+		this._registeredEvents.push(...events);
```
Comment (shuse2) [inline framework/src/controller/ws/ws_server.ts:45]: removed registration in the start, and being registered separately due to the initialization order

File: framework/src/controller/bus.ts [hunk 2]
```diff
@@ -178,8 +179,15 @@ export class Bus {
 										);
 									});
 							})
-							.catch(err => {
-								this.logger.debug(err, 'Error occurred while sending RPC results.');
+							.catch((err: JSONRPCError) => {
+								rpcServer
+									.send([sender, requestData.id as string, JSON.stringify(err.response)])
```
Comment (shuse2) [inline framework/src/controller/bus.ts:184]: Fixing when error happens, it should return error response to the client

File: elements/lisk-client/package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
 		"prestart": "./scripts/prestart.sh",
 		"start": "./scripts/start.sh",
 		"start:ws_server": "node cypress/ws_server.js",
-		"browserify": "browserify ./dist-node/index.js -o ./dist-browser/index.js -s lisk",
+		"browserify": "browserify ./dist-node/index.js -o ./dist-browser/index.js -s lisk --ignore zeromq",
```
Comment (shuse2) [inline elements/lisk-client/package.json:33]: browser version needs to ignore zeromq

File: elements/lisk-api-client/src/ws_channel.ts [hunk 4]
```diff
@@ -165,10 +134,33 @@ export class WSChannel {
 	}
 
 	public subscribe<T = Record<string, unknown>>(eventName: string, cb: EventCallback<T>): void {
+		const request = {
+			jsonrpc: '2.0',
+			id: this._requestCounter,
+			method: 'subscribe',
+			params: {
+				topics: [eventName],
```
Comment (ishantiw) [inline elements/lisk-api-client/src/ws_channel.ts:142]: topics is an array, is it possible to subscribe to multiple events at once? if not, then maybe we can keep it singular topic as a string?

**Review by ishantiw — APPROVED** [review]
Looks good, just a small comment

**Review by mitsuaki-u — APPROVED** [review]
Other than @ishantiw comment, LGTM

File: elements/lisk-api-client/src/ws_channel.ts (hunk 4 above)
Comment (shuse2) [reply elements/lisk-api-client/src/ws_channel.ts:142]: yes, topics is array, and in theory it's possible to subscribe multiple at once. right now it's just not allowing per interface, but maybe we could improve it later?

**Pull request merged**